qiskit/pulse/schedule.py

```diff
@@ -769,17 +789,24 @@ def __init__(self, *schedules, parameters: Optional[Dict[str, Union[float, compl
         parameterized = []
         parameters = parameters or []
         self.name = name or ''
-        # partition schedules into callable and schedules
-        for schedule in schedules:
-            if isinstance(schedule, ParameterizedSchedule):
-                parameterized.append(schedule)
-                parameters += schedule.parameters
-            elif callable(schedule):
-                parameterized.append(schedule)
-            elif isinstance(schedule, Schedule):
-                full_schedules.append(schedule)
-            else:
-                raise PulseError('Input type: {} not supported'.format(type(schedule)))
+
```

Comment:
I wasn't sure about this, but I supposed you wanted to deprecate the argument for `ParameterizedSchedule` too. Just let me know if that's not the case, and I'll revert the change.